src/presentation/http/router/noteSettings.test.ts

```diff
@@ -11,10 +11,22 @@ describe('NoteSettings API', () => {
 
       const expectedNoteSettings = {
         'customHostname': 'codex.so',
-        'isPublic': true,
         'id': 2,
-        'noteId': 2,
         'invitationHash': 'E2zRXv3cp-',
+        'isPublic': true,
+        'noteId': 2,
+        'team':  [
```

Comment:
I think it's better not to touch existing test but create the new one that will check that if note has a team, it is returning with note settings